src/StorageClient.ts

```diff
@@ -15,3 +15,6 @@ export class SupabaseStorageClient extends StorageBucketApi {
     return new StorageFileApi(this.url, this.headers, id, this.fetch)
   }
 }
+
+// Alias `StorageClient` as `SupabaseStorageClient` for backwards compatibility.
+export { StorageClient as SupabaseStorageClient }
```

Comment:
Should have noticed this earlier. You'll also need to change the import in `src/index.ts`. This export could be moved to there as well.

Reply:
yep, i was already looking into that